style.css

```diff
@@ -98,6 +98,11 @@ p {
 a {
 	color: white;
 	text-decoration: none;
+	font-size: 24px;
```

Comment:
Is there a specific set of links that we are targeting here? We should use a more specific selector, rather than setting the font size on every link.

Reply:
Added .top-links li a {
font-size: 24px}     to target just the links in the header